src/Root.jsx

```diff
@@ -7,6 +7,18 @@ import { Web3ContextProvider } from "./hooks/web3Context";
 import App from "./App";
 import store from "./store";
 
+// Google Analytics
+import ReactGA from "react-ga4";
+const TRACKING_ID = "G-TKEFGWGLPM";
+ReactGA.initialize(TRACKING_ID);
+//
+// Initialize google analytics page view tracking
+import { createBrowserHistory } from "history";
+const gaHistory = createBrowserHistory();
```

Comment:
Do we not need to instantiate `Router history=history`? 
This just works?

Reply:
it works. I tested with my own GA4 key.

Reply:
narrator: it didn't just work. https://github.com/OlympusDAO/olympus-frontend/pull/372/commits/5dd6ad555a9a85dcafd75d7e813322bff39bb953 moved the code into `src/hooks/analytics` with a custom hook to make it all work properly.